### PR TITLE
docs(dotnet): Frame examples

### PR DESCRIPTION
**Pull request by avodovnik**

### CODE REVIEW

File: docs/src/api/class-frame.md [hunk 1]
```diff
@@ -97,6 +97,33 @@ with sync_playwright() as playwright:
     run(playwright)
 ```
 
+```csharp
+using Microsoft.Playwright;
+using System;
+using System.Threading.Tasks;
+
```
Comment (kblok) [inline docs/src/api/class-frame.md:104]: Do you think we need to add the using in examples? just wondering.

File: docs/src/api/class-frame.md [hunk 2]
```diff
@@ -97,6 +97,33 @@ with sync_playwright() as playwright:
...
+{
+    public static async Task Main()
+    {
+        using var playwright = await Playwright.CreateAsync();
+        var browser = await playwright.Firefox.LaunchAsync();
+        var page = await browser.NewPageAsync();
+
+        await page.GoToAsync("https://www.bing.com");
+        DumpFrameTree(page.MainFrame, string.Empty);
+        await browser.CloseAsync();
```
Comment (kblok) [inline docs/src/api/class-frame.md:115]: Why not doing `await using`? we would be teaching people new stuff :)

File: docs/src/api/class-frame.md (hunk 1 above)
Comment (kblok) [reply docs/src/api/class-frame.md:104]: This applies to the rest of the open PRs

File: docs/src/api/class-frame.md (hunk 2 above)
Comment (kblok) [reply docs/src/api/class-frame.md:115]: This applies to the rest of the open PRs

File: docs/src/api/class-frame.md (hunk 1 above)
Comment (avodovnik) [reply docs/src/api/class-frame.md:104]: I mimic the other languages. Some examples are "full", and it feels natural for us to do the same. *Ideally* down the road, we'll have a "Run this example" somewhere, but... until then... I'm up for consistency.

File: docs/src/api/class-frame.md (hunk 2 above)
Comment (avodovnik) [reply docs/src/api/class-frame.md]: Yup, I've changed all three occurrences.

File: docs/src/api/class-frame.md [hunk 3]
```diff
@@ -421,6 +463,10 @@ divs_counts = await frame.eval_on_selector_all("div", "(divs, min) => divs.lengt
 divs_counts = frame.eval_on_selector_all("div", "(divs, min) => divs.length >= min", 10)
 ```
 
+```csharp
+var divsCount = await page.EvalOnSelectorAllAsync<bool>("div", "(divs, min) => divs.length >= min", 10);
```
Comment (avodovnik) [inline docs/src/api/class-frame.md:467]:
```suggestion
var divsCount = await frame.EvalOnSelectorAllAsync<bool>("div", "(divs, min) => divs.length >= min", 10);
```

**Pull request merged**